src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Controller.cs

```diff
@@ -77,14 +77,26 @@ internal override void OnModelUpdated(Model modelOpt)
             }
             else
             {
-                var quickInfoItem = modelOpt.Item;
-
                 // We want the span to actually only go up to the caret.  So get the expected span
                 // and then update its end point accordingly.
-                var triggerSpan = modelOpt.GetCurrentSpanInSnapshot(quickInfoItem.TextSpan, this.SubjectBuffer.CurrentSnapshot);
-                var trackingSpan = triggerSpan.CreateTrackingSpan(SpanTrackingMode.EdgeInclusive);
+                ITrackingSpan trackingSpan;
+                QuickInfoItem item = null;
```

Comment:
Doesn't this end up being equal to modelOpt.Item? Your line 88 is assuming modelOpt is non-null, and you assign it in that case